### ISSUE TITLE
Unable to build LDDs for 1E00

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/pds4-information-model#776]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

When I did tried to build an LDD with 1E00, it fails.

### 🕵️ Expected behavior

I expected the LDDs to be generated

### 📜 To Reproduce

```bash
lddtool-15.2.0-SNAPSHOT/bin/lddtool -V 1E00 -lpJ ../ldd-ctli/src/PDS4_CTLI_IngestLDD.xml
```
outputs:
```
>>  ERROR 11179 data dictionary class is missing for overwrite - Identifier:ObjectClass.OC.0001_NASA_PDS_1.pds.Cartography_Group
>>  ERROR 11179 data dictionary class is missing for overwrite - Identifier:ObjectClass.OC.0001_NASA_PDS_1.pds.Rings
...
```

### 🖥 Environment Info

Mac OSx

### 📚 Version of Software Used

15.1.0

### 🩺 Test Data / Additional context

https://github.com/pds-data-dictionaries/ldd-ctli/blob/main/src/PDS4_CTLI_IngestLDD.xml

**Comment by jordanpadams** [comment]
Fixed per #775

**Comment by jordanpadams** [comment]
Skipping I&T because this is tested over and over with smoke tests
